### PR TITLE
docs: encourage people to use latest stable version

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

People are often asking about issues that have been fixed in current stable.

## How This PR Solves The Issue

* Encourage them to upgrade before filing an issue.
* Minor improvements to checkboxes and suggestions to start with.
* Minor improvements to `ddev debug test`, including curl of google.com to test internet access
* Minor improvements to troubleshooting suggestions

Here's what it looks like:

image

### CODE REVIEW

File: cmd/ddev/cmd/scripts/test_ddev.sh [hunk 1]
```diff
@@ -30,6 +30,13 @@ function docker_desktop_version {
   fi
 }
 
+header "Please make sure that you have already looked at troubleshooting guide"
+
+printf "Troubleshooting guide: https://ddev.readthedocs.io/en/stable/users/usage/troubleshooting/ \n"
+printf "Simple things to check:\n* ddev poweroff\n* Restart Docker Provider\n* Reboot computer\n* Temporarily disable VPN and firewall\n"
+printf "Press any key to continue:\n"
+read x
```
Comment (stasadev) [inline cmd/ddev/cmd/scripts/test_ddev.sh:38]:
```suggestion
[[ "${DDEV_NONINTERACTIVE:-}" == "true" ]] && read x
```

**Review by stasadev — APPROVED** [review]
Looks good to me!

A small suggestion with `$DDEV_NONINTERACTIVE`, otherwise `TestCmdDebugTest` will get stuck.

**Review by stasadev — APPROVED** [review]
Looks good to me!

A small suggestion with `$DDEV_NONINTERACTIVE`, otherwise `TestCmdDebugTest` will get stuck.

File: cmd/ddev/cmd/scripts/test_ddev.sh (hunk 1 above)
Comment (rfay) [reply cmd/ddev/cmd/scripts/test_ddev.sh:38]: Thanks for watching that carefully!

**Pull request merged**